packages/bloc_dispose_scope/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: bloc_dispose_scope
 description: Package supporting bloc types for dispose_scope.
-version: 3.0.0
+version: 3.1.0
```

Comment:
Let's make it a patch bump instead, unless you have a strong argument for it to be a minor update, as I'm on the fence about this